rest/api.go

```diff
@@ -39,7 +39,7 @@ type rootResponse struct {
 	CouchDB          string `json:"couchdb,omitempty"` // TODO: Lithium - remove couchdb welcome
 	Vendor           vendor `json:"vendor,omitempty"`
 	Version          string `json:"version,omitempty"`
-	PersistentConfig bool   `json:"persistentConfig,omitempty"`
+	PersistentConfig bool   `json:"persistent_config,omitempty"`
```

Comment:
I missed one thing in my last review - `omitempty` on a bool will remove the field in the response if the value is `false`, which ideally we'd not want to do here.

```suggestion
	PersistentConfig bool   `json:"persistent_config"`
```